cmd/node/config/config.toml

```diff
@@ -281,8 +281,8 @@
 
 # TODO: change this config to real numbers before merging to development
 [EpochStartConfig]
-    MinRoundsBetweenEpochs = 1000000
-    RoundsPerEpoch         = 1000000
+    MinRoundsBetweenEpochs = 100
```

Comment:
change it to 500 and 600

Reply:
ok, so the development will have epoch changes. wouldn't it be better if we leave high numbers until we fix all issues relates to epochs ?

Reply:
changed back to 1000000